docs/architecture/modernize-desktop/deploy-modern-applications.md

```diff
@@ -0,0 +1,220 @@
+
+Installing desktop applications have traditionally been a nightmare for two main reasons:
+
+- Lack of close collaboration culture between dev and IT teams.
+- Lack of a solid packaging and deploying technology we can build upon.
+
+In fact, we've been living with the fact that sometimes you regret that you installed an app because:
+
+- It ends up having some undesired side effects on your machine.
+- Some applications that were previously installed stop working.
+
+Additionally, you can't just restore the system to its original state by uninstalling the app. We're so used to live this situation that we've coined terms like "DLL Hell" or "Winrot".
+
+In this chapter, we'll talk about MSIX. MSIX is the brand-new technology from Microsoft that tries to capture the best of previous technologies to provide a solid foundation for the packaging technology of the future.
+
+What does a packaging technology have to do with modernization? Well, it turns out that packaging is fundamental for the enterprise IT with lots of money invested there. Modernization isn't only related to using the latest technologies. It's also related to reducing time to market from the moment a business requirement is defined until your company delivers the feature to your client.
+
+## The modern application lifecycle
+
+Today, developers write and build the code for an app and then pass the generated assets to the IT Pros. Then, the IT Pros reconfigure the app and repackage it, typically in an MSI or more recently in an App-V packaging format. The app is then deployed through different channels and tools. One of the main problems with this approach is commonly known as "packaging paralysis". The problem is that this cycle repeats every time there's an app update or an OS update.
+
+You can see the process reflected on the following picture:
+
+![Diagram showing the modern IT lifecyle](./media/deploying-modern-desktop-applications/modern-it-application-lifecycle.png)
+
+Companies need a way to break this packaging cycle into three independent cycles:
+
+- OS updates
+- Application updates
+- Customization
+
+![Diagram showing the modern IT virtuous cycles](./media/deploying-modern-desktop-applications/modern-it-virtuous-cycles.png)
```

Comment:
Rename the images folder to match the new article file name and fix its references. e.g.

```suggestion
![Diagram showing the modern IT virtuous cycles](./media/deploy-modern-applications/modern-it-virtuous-cycles.png)
```